checker/src/main/java/org/checkerframework/checker/nullness/qual/EnsuresMultiNonNull.java

```diff
@@ -0,0 +1,28 @@
+package org.checkerframework.checker.nullness.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.InheritedAnnotation;
+import org.checkerframework.framework.qual.PostconditionAnnotation;
+
+/**
+ * An annotation wrapper to allow multiple {@code @EnsuresNonNull} annotation at the same location.
+ *
+ * <p>Programmers generally do not need to use this; it is created by Java when a programmer writes
+ * more than one {@code @EnsuresNonNull} annotation at the same location.
+ *
+ * @see EnsuresNonNull
+ * @checker_framework.manual #java-expressions-as-arguments Syntax of Java expressions
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
+@PostconditionAnnotation(qualifier = NonNull.class)
+@InheritedAnnotation
+public @interface EnsuresMultiNonNull {
+
+    EnsuresNonNull[] value();
```

Comment:
Please write a Javadoc comment on every annotation element (field).

Reply:
Is this Ok?